code-studio/src/js/code-studio.js

```diff
@@ -27,7 +27,8 @@ require('./components/report_abuse_form');
 require('./components/send_to_phone');
 require('./components/small_footer');
 require('./components/GridEditor');
-require('./components/IconLibrary');
+require('./components/ImagePicker');
+require('./components/Attachments');
```

Comment:
It looks like `ImagePicker` doesn't actually need to placed on window.dashboard.